Documentation/services/pcn-packetcapture/packetcapture.rst

```diff
@@ -1,14 +1,14 @@
 Packetcapture service
 =====================
```

Comment:
It would be helpful to add another rst file with the overview of the implementation, i.e., how you can leverage libpcap syntax for filtering packets in eBPF.

Reply:
Something like this one https://blog.cloudflare.com/xdpcap/ ?
or maybe just a reference to the link above in the actual documentation?